netweaver/setup/swap_space.sls

```diff
@@ -0,0 +1,37 @@
+{%- from "netweaver/map.jinja" import netweaver with context -%}
+{% set host = grains['host'] %}
+
+{% for node in netweaver.nodes if host == node.host %}
```

Comment:
I would say this for loop is not needed as this operation must be applied to all nodes (actually, this is what will happen with the current implementation, but it's more confusing)